src/Files.App/Helpers/AppThemeResourcesHelper.cs

```diff
@@ -25,22 +25,22 @@ public static void LoadAppResources(this IResourcesService service, IAppearanceS
 			service.SetCompactSpacing(useCompactStyles);
 			service.SetAppThemeBackgroundColor(appThemeBackgroundColor.FromWindowsColor());
 
-			if (!String.IsNullOrWhiteSpace(appThemeAddressBarBackgroundColor) && appThemeAddressBarBackgroundColor != "#00000000")
+			if (!string.IsNullOrWhiteSpace(appThemeAddressBarBackgroundColor) && appThemeAddressBarBackgroundColor != "#00000000")
 				service.SetAppThemeAddressBarBackgroundColor(ColorHelper.ToColor(appThemeAddressBarBackgroundColor).FromWindowsColor());
 			else
 				appearance.AppThemeAddressBarBackgroundColor = ""; //migrate to new default
 
-			if (!String.IsNullOrWhiteSpace(appThemeSidebarBackgroundColor) && appThemeAddressBarBackgroundColor != "#00000000")
+			if (!string.IsNullOrWhiteSpace(appThemeSidebarBackgroundColor) && appThemeAddressBarBackgroundColor != "#00000000")
 				service.SetAppThemeSidebarBackgroundColor(ColorHelper.ToColor(appThemeSidebarBackgroundColor).FromWindowsColor());
 			else
 				appearance.AppThemeSidebarBackgroundColor = ""; //migrate to new default
 
-			if (!String.IsNullOrWhiteSpace(appThemeFileAreaBackgroundColor) && appThemeAddressBarBackgroundColor != "#00000000")
+			if (!string.IsNullOrWhiteSpace(appThemeFileAreaBackgroundColor) && appThemeAddressBarBackgroundColor != "#00000000")
 				service.SetAppThemeFileAreaBackgroundColor(ColorHelper.ToColor(appThemeFileAreaBackgroundColor).FromWindowsColor());
 			else
 				appearance.AppThemeFileAreaBackgroundColor = ""; //migrate to new default
 
-			if (appThemeFontFamily != "Segoe UI Variable")
+			if (appThemeFontFamily != Constants.Appearance.StandardFont)
```

Comment:
I typically only create a constant if it's going to be reused

Reply:
Using constants to reduce hard coding is a good reason.

Reply:
The reasoning is indeed to reduce hard coding.

Reply:
Should it at least be local to this helper?